flows/nodes/attach.md

```diff
@@ -0,0 +1,45 @@
+# Attach
+
+The Attach node connects to an existing Gravwell query, specified by a search ID. Like the [Run Query](runquery) node, it outputs a structure into the payload (named `search` by default) which contains information about the search and allows other nodes to access the results.
+
+The most common use of the Attach node is to connect to the scheduled search which triggered the [Alert](/alerts/alerts) for which the current flow is a consumer. See the example below for more information on this use-case.
```

Comment:
```suggestion
The most common use of the Attach node is to connect to the scheduled search which triggered the [Alert](/alerts/alerts) for which the current flow is a consumer. See the example below for more information on this use case.
```

I don't think we need a hyphen here and I don't think I typically see it hyphenated on other sites.